analytics-api/tests/conftest.py

```diff
@@ -82,7 +82,7 @@ def session(app, db):  # pylint: disable=redefined-outer-name, invalid-name
         txn = conn.begin()
 
         options = dict(bind=conn, binds={})
-        sess = db.create_scoped_session(options=options)
+        sess = db._make_scoped_session(options=options)
```

Comment:
It looks like they made this method private. I'm curious, is there a different one that developers need to use to create a scoped session?

Reply:
Updated it, thanks for noting this.